common/tier4_vehicle_rviz_plugin/src/tools/turn_signal.cpp

```diff
@@ -18,22 +18,32 @@
 #include <ament_index_cpp/get_package_share_directory.hpp>
 #include <rviz_common/uniform_string_stream.hpp>
 
+#include <X11/Xlib.h>
+
 namespace rviz_plugins
 {
 TurnSignalDisplay::TurnSignalDisplay()
 {
+  const Screen * screen_info = DefaultScreenOfDisplay(XOpenDisplay(NULL));
+
+  const bool large_screen = screen_info->height > 2000;
+  const auto left = large_screen ? 196 : 98;
+  const auto top = large_screen ? 350 : 175;
+  const auto width = large_screen ? 512 : 256;
+  const auto height = large_screen ? 256 : 128;
```

Comment:
Rather than switching between FHD or 4K, it would be better to fix the ratio.

Reply:
addressed in [6dc12a9](https://github.com/autowarefoundation/autoware.universe/pull/587/commits/6dc12a92b61fdcbe6ae881c752ff6e0977675460)